vng_api_common/templates/vng_api_common/index.html

```diff
@@ -36,7 +36,7 @@ <h5 class="card-title"><i class="fas fa-wifi"></i> API specificatie</h5>
               <div class="card-body">
                 <h5 class="card-title"><i class="fas fa-shield-alt"></i> API scopes</h5>
                 <p class="card-text">Alle scopes binnen deze API voor gebruik in JWT's t.b.v. autorisaties.</p>
-                <a href="{% url 'vng_api_common:scopes' %}" class="btn btn-primary">API scopes</a>
+                <a href="https://github.com/{{ settings.SELF_REPO }}/blob/{{ settings.API_VERSION }}/src/scopes.md" class="btn btn-primary">API scopes</a>
```

Comment:
Hmm, maar dit gaat voor componenten op de test omgeving niet goed, want daar wordt "master" gedeployed.

Reply:
Maar ook bij de testcomponenten is er een `API_VERSION` toch? Of wil je dat hij dan echt master pakt, i.p.v. de laatste tag?

Reply:
De test domeinen draaien `master` toch?

De master branch kan natuurlijk wel versie 2.0.0 zijn voor intern, maar maar dit is sowieso geen tag (en dat neem je wel aan in je URL). De URL moet dus ook wel degelijk naar master (kunnen) wijzen terwijl de API versie bijvoorbeeld 2.0.0-dev is.

Reply:
Ik heb het nu zo gemaakt dat `settings.SELF_BRANCH` te zetten is via een envvar, en als default `settings.API_VERSION` gebruikt (https://github.com/VNG-Realisatie/gemma-documentregistratiecomponent/pull/132/files#diff-ec32e62a1606fceb1cf06b04676170492c4c7a93374a1c5ea6d29b2337dccd49R53)